File: try/deno/denopython/class1.js
import { python } from "https://deno.land/x/python/mod.ts";

const { Person } = python.runModule(`
class Person:
  def __init__(self, name):
    self.name = name
`);

const person = new Person("John");

console.log('person.name.valueOf()=', person.name.valueOf());
console.log('person.name = "Jane"');
console.log('person.name.valueOf()=', person.name.valueOf());
console.log('name in person=', "name" in person);

File: try/dart/class1.dart
class Person {
    String name = '';
    int age = 0;

    Person(String name, int age) {
        this.name = name;
        this.age = age;
    }

    @override
    String toString() {
        return this.name+' '+this.age.toString();
    }
}

void main() {
    print(Person);
    var p=new Person('ccc', 52);
    print(p);
    print(p.toString());
}
